src/Microsoft.ComponentDetection.Detectors/rust/Contracts/CargoLock.cs

```diff
@@ -1,15 +1,19 @@
-﻿using System.Diagnostics.CodeAnalysis;
-using Nett;
+using System.Diagnostics.CodeAnalysis;
 
 namespace Microsoft.ComponentDetection.Detectors.Rust.Contracts
 {
+    using System.Collections.Generic;
+    using System.Runtime.Serialization;
+    using Tomlyn.Model;
+
     // Represents Cargo.Lock file structure.
+    [DataContract]
     public class CargoLock
     {
-        [SuppressMessage("StyleCop.CSharp.NamingRules", "SA1300:ElementMustBeginWithUpperCaseLetter", Justification = "Deserialization contract. Casing cannot be overwritten.")]
-        public CargoPackage[] package { get; set; }
+        [DataMember(Name = "package")]
+        public List<CargoPackage> Package { get; set; }
 
-        [SuppressMessage("StyleCop.CSharp.NamingRules", "SA1300:ElementMustBeginWithUpperCaseLetter", Justification = "Deserialization contract. Casing cannot be overwritten.")]
-        public TomlTable metadata { get; set; }
+        [DataMember(Name = "metadata")]
+        public TomlTable Metadata { get; set; }
```

Comment:
Can you be more specific than `TomlTable` here?

Reply:
Would List<object> be better?